addon/locales/en/components/map-commands-dialogs/go-to.js

```diff
@@ -2,7 +2,8 @@ export default {
   'caption': 'Go to specified point',
   'error-message': {
     'caption': 'Go to error',
-    'content': 'Specified coordinates are wrong & can\'t be successfully converted into numbers'
+    'content': 'Specified coordinates are wrong & can\'t be successfully converted into numbers',
+    'large-coords': 'Specified coordinates doesn\'t exist in allowable map area'
```

Comment:
Same problem in the resulting popup:
![image](https://cloud.githubusercontent.com/assets/7602229/26236598/99deb6da-3c8a-11e7-957e-9503b6874660.png)